xaitk_saliency/utils/masking.py

```diff
@@ -28,3 +81,32 @@ def generate_masked_images(
         masked_img = np.multiply(mask, img, casting='unsafe')
         masked_imgs.append(PIL.Image.fromarray(np.uint8(masked_img)))
     return masked_imgs
+
+
+def weight_regions_by_scalar(
+    scalar_vec: np.ndarray,
+    masks: np.ndarray
+) -> np.ndarray:
+    """
+    Weight some binary masks reguib with its respective vector in scalar_vec.
```

Comment:
"reguib" is meant to be "region" right? (not sure if this is some specialized XAI terminology or a typo)

Reply:
Yes, that was a typo. Thank you for catching that!